docker.sh

```diff
@@ -33,12 +31,11 @@ COMMIT=$(git rev-parse HEAD)
 echo "COMMIT=\"${COMMIT}\"" > indexd/index/version_data.py
 
 BUILD_COMMAND=(build \
-  --label org.opencontainers.image.version="${VERSION}" \
```

Comment:
This is to get rid of "1.2.3.devN" as version, right?

Reply:
yes